src/System.ComponentModel.TypeConverter/tests/Drawing/FontConverterTests.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Drawing;
+using Xunit;
+
+namespace System.ComponentModel.TypeConverterTests
+{
+    public class FontNameConverterTest
+    {
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TestConvertFrom()
+        {
+            FontConverter.FontNameConverter converter = new FontConverter.FontNameConverter();
+            // returns "Times" under Linux and "Times New Roman" under Windows
+            Assert.True((converter.ConvertFrom("Times") as string).StartsWith("Times"), "string test");
```

Comment:
We can do this startsWith or we can also have a Linux and a Windows specific test. What if someone broke Windows converter and returned Times New Roman3 instead? This test will not fail. So please split it into two tests using 

`[PlatformSpecific(TestPlatforms.AnyUnix)]` (this are the available [TestPlatforms](https://github.com/dotnet/buildtools/blob/master/src/xunit.netcore.extensions/TestPlatforms.cs)) or even better I think you could use [`PlatformDetection.IsWindows`](https://github.com/dotnet/corefx/blob/master/src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Unix.cs#L17) to detect if we're running on Windows and assert the expected value depending on the running OS.

Reply:
When I did this, the test started failing. So I realized, that converter taken from Mono [just returned input value](https://github.com/mono/mono/blob/master/mcs/class/System.Drawing/System.Drawing/FontConverter.cs#L344). So I changed it according to what is in the [full .NET framework](https://referencesource.microsoft.com/#System.Drawing/commonui/System/Drawing/FontConverter.cs,637).